lib/dark_worlds_server/runner_supervisor/runner.ex

```diff
@@ -229,29 +228,12 @@ defmodule DarkWorldsServer.RunnerSupervisor.Runner do
   end
 
   def handle_info({:spawn_bots, bot_count}, state) when bot_count > 0 do
```

Comment:
I think the idea is that bots are not some kind of special player but should behave like a regular player, do we need this handle_info for some special reason? 🤔

Reply:
Now that you say it, this could go in `matching_coordinator.ex` so I'll move it

Reply:
Done in bb8dee8